src/main/Controls.js

```diff
@@ -24,6 +24,7 @@ class Controls extends React.Component<Props> {
 
   constructor(props: Object) {
     super(props);
+    this.state = {defaultHalfInterval:2};
```

Comment:
what is defaultHalfInterval? Can you add a comment here to define what it is?

Reply:
Done.